### PR TITLE
Adding a draft for models.

### PR DESCRIPTION
**Pull request by gabelula**
To discuss.

### CODE REVIEW

File: models/asset.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+var mongoose = require('mongoose');
```
Comment (wyattjoh) [inline models/asset.js]: Should import the mongoose that's in the current folder, ie:

``` js
const mongoose = require('../mongoose');
```

Also, should use const wherever possible, never use var.

**Review by wyattjoh — CHANGES_REQUESTED** [review]
Should use `const/let` instead of `var` in ES6. Also don't import raw `mongoose`, instead grab the one from the mongoose file as in the comments listed below.

File: models/asset.js (hunk 1 above)
Comment (gabelula) [reply models/asset.js]: ok. Done!

**Review by wyattjoh — APPROVED** [review]

**Review by wyattjoh — CHANGES_REQUESTED** [review]
Foreign keys are not referenced using models. Decided to hold off until resolved.

**Comment by gabelula** [comment]
I wonder if we should also add "children" to the comment's model (we have parent right now).

File: models/comments.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
+'use strict';
```
Comment (okbel) [inline models/comments.js]: This file must be `comment.js` instead of `commentS.js`

**Comment by gabelula** [comment]
Anything else to be able to merge this one? @wyattjoh @okbel @jde

File: models/comments.js (hunk 2 above)
Comment (gabelula) [reply models/comments.js]: Done

File: models/action.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    createdAt: 'created_at',
+    updatedAt: 'updated_at'
+  }
+});
+
+/**
+ * Finds an action by the id.
+ * @param {String} id  identifier of the action (uuid)
+*/
+ActionSchema.methods.findById = function(id) {
```
Comment (riley) [inline models/action.js]: this should be a static e.g. `ActionSchema.statics.findById = function (id) {`

**Review by riley — CHANGES_REQUESTED** [review]
just one little thing I see

**Review by riley — APPROVED** [review]

**Review by wyattjoh — APPROVED** [review]

**Pull request merged**